burotel/indico_burotel/template_overrides/core/rb/emails/reservations/reminders/finishing_bookings.html

```diff
@@ -0,0 +1,18 @@
+{% extends '~rb/emails/reservations/reminders/finishing_bookings.html' %}
+
+{% block subject %}Some of your desk reservations are finishing soon{% endblock %}
```

Comment:
Do we want to go for "bookings" instead of "reservations" for user-visible stuff?

Reply:
Hm, did we agree on this somewhere in Indico?

Reply:
I don't mind changing it but I have an impression that we use it interchangeably even on the backend.

Reply:
the old code usually uses reservation, the new code usually uses booking